aptos-move/move-examples/mint_nft/sources/minting.move

```diff
@@ -0,0 +1,277 @@
+/// This module is an example of how one can create a NFT collection from a resource account
```

Comment:
we don't have license, is that normal for move?

Reply:
I global search'ed `// Copyright (c) Aptos
// SPDX-License-Identifier: Apache-2.0
` in move files, don't think I saw it anywhere...